client/src/main/java/org/apache/falcon/client/FalconClient.java

```diff
@@ -1169,7 +1169,7 @@ private JSONObject getExtensionJobDetailJson(String jobName) {
     private List<Entity> getEntities(String extensionName, String jobName, InputStream configStream,
                                      String extensionType, String extensionBuildLocation) {
         List<Entity> entities = null;
-        if (!extensionType.equals(ExtensionType.CUSTOM.name())) {
+        if (!extensionType.equals(ExtensionType.TRUSTED.toString())) {
```

Comment:
This comes as Trusted extension from the server as its .toString while coming from the server.